serveradmin/serverdb/models.py

```diff
@@ -151,6 +166,29 @@ def from_str(self, value):
         except ValueError as error:
             raise ValidationError(str(error))
 
+    def get_compiled_regexp(self):
```

Comment:
Shouldn't this be `_get_compiled_regexp`, or maybe `@property _compiled_regexp`?